includes/components/class-elementor.php

```diff
@@ -115,6 +115,12 @@ public function add_icon_styles() {
 				height: 28px;
 				background: url("' . esc_url( $icon_url ) . '") center center no-repeat;
 				background-size: 28px;
+			}
+			@media screen and (prefers-color-scheme: dark) {
+				.eicon-hivepress {
+					background: url("' . hivepress()->get_url() . '/assets/images/logo-light.svg") center center no-repeat;
```

Comment:
It seems that there's no need to repeat some styles, e.g. background size, position for background - these will be inherited, so background-image should be enough. You can also get rid of the variable above, it's used once.